CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## v2.0.0 - 2022-09-22
+
+## Added
+
+- #109 - Migration to use `python-ipfabric` library as the API client to IP Fabric.
+- #103 - New command `get-loaded-snapshots` for greater snapshot details.
+
+## Changed
+
+- Version 2.0.0 of `nautobot-plugin-chatops-ipfabric` only supports IP Fabric version v5.0
```

Comment:
and higher I guess

Reply:
From what I gathered I think each major release on their SDK/OS will require a major release for this plugin. I may have misunderstood though.

Reply:
correct

Reply:
updated